styleguide/components/DemoLayoutNav.es6.js

```diff
@@ -15,7 +15,7 @@ function DemoLayoutNav ({ children }: DemoLayoutNavProps) {
       </a>
       <div className="app__nav-intro">
         <h1>Design System</h1>
-        <p>Welcome to B12 Design language! Here you'll find all available components that are used to build our awesome product.</p>
+        <p>Welcome to B12 Design language! Here you&apos;ll find all available components that are used to build our awesome product.</p>
```

Comment:
Same here: I think it will print `&apos;`

Reply:
![image](https://user-images.githubusercontent.com/35793666/177470854-4ca2604f-8a18-4894-bd6b-d7aa1b7082c9.png)